src/CodeGen_LLVM.cpp

```diff
@@ -1119,6 +1119,13 @@ void CodeGen_LLVM::optimize_module() {
     llvm::CGSCCAnalysisManager cgam;
     llvm::ModuleAnalysisManager mam;
 
+#if LLVM_VERSION < 140
```

Comment:
Note that pedantically, this check is wrong for the first half of commits between 14-init and 14.0.0,
since i believe it says that it's not needed starting with 14-init, while it's not needed starting with 14-init+~2k.